### DISCUSSION TITLE
Edit motion file

### DISCUSSION THREAD

**Discussion by techmech46** [cyberbotics/webots#4120] [Q&A]
I am using WEBOT with NAO. Currently, my robot walk for 10 steps by pressing forward button once. But, I want to make it to only 6 steps.
For that do I need to edit the forwards.motion file?
If yes then how can I edit and execute the task.
If no, then how can I limit it to only 6 or less steps.

Thank you in advance

**Comment by omichel** [comment]
Motion files are text files which you can edit with a text editor. Although, it may be tricky to implement a complex motion, it should be easy to remove steps from a motion file, by simply removing lines.
